tests/frontend/test_configuration.py

```diff
@@ -131,6 +131,148 @@ def test_conf_file_loads_well(self, monkeypatch, tmpdir):
 
         assert configuration == EXPECTED_CONFIG
 
+    def test_get_api_section_safely_error(self, monkeypatch, tmpdir, caplog):
+        """Test that the get_api_section_safely function raises an error and
+        logs correctly if there is no api section in the configuration file."""
+        filename = tmpdir.join("config.toml")
+
+        empty_file = ""
+
+        with open(filename, "w") as f:
+            f.write(empty_file)
+
+        with monkeypatch.context() as m:
+            with pytest.raises(conf.ConfigurationError, match=""):
+                m.setattr(os, "getcwd", lambda: tmpdir)
+                configuration = conf.load_config()
+
+        assert "does not contain an \"api\" section" in caplog.text
```

Comment:
I think there's an issue with string escaping in the error message generated. The error message ends with
```python
"file does not \'contain an "api" section.\'\n'"
```

Reply:
(note the additional escaped inverted commas)